catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java

```diff
@@ -296,12 +297,10 @@ private SourceResponse executePostFederationQueryPlugins(
 
     try {
       for (PostFederatedQueryPlugin service : postQuery) {
-        try {
-          queryResponse = service.process(queryResponse);
-        } catch (PluginExecutionException e) {
-          LOGGER.info("Error executing PostFederatedQueryPlugin", e);
-        }
+        queryResponse = service.process(queryResponse);
```

Comment:
❗️ There is a change in behavior here. Before it would continue looping to all other plugins where as now it stops at the first failure. Was that what we wanted?